plaidbench.py

```diff
@@ -257,30 +260,65 @@ def main():
         outputs['run_configuration'] = data.copy()
     else:
         networks.append(args.module)
+        batch_list.append(args.batch_size)
 
     for network in networks:
         printf("\nCurrent network being run : " + network)  
         args.module = network
         network_data = {}
+    
 
-        # Run network
-        try:
-            value_check(examples, epochs, batch_size)
-
-            # Setup
-            x_train, y_train = setup(args.train, epoch_size, batch_size)
+        for batch in batch_list:
+            batch_size = batch
+            printf('running with a batch size of ' + str(batch_size))
+        
+            # Run network w/ batch_size
+            try:
+                value_check(examples, epochs, batch_size)
+
+                # Setup
+                x_train, y_train = setup(args.train, epoch_size, batch_size)
+
+                # Loading the model
+                module, x_train, model = load_model(args.module, x_train)
+
+                if args.train:
+                    # training run
+                    train(x_train, y_train, epoch_size, model, batch_size, 
+                          compile_stop_watch, epochs, stop_watch, output, network)
+                else:
+                    # inference run
+                    inference(args.module, model, batch_size, compile_stop_watch, 
+                              output, x_train, examples, stop_watch)
+
+                # Record stopwatch times
+                execution_duration = stop_watch.elapsed()
+                compile_duration = compile_stop_watch.elapsed()
+
+                network_data['compile_duration'] = compile_duration
+                network_data['execution_duration'] = execution_duration / examples
+                network_data['precision'] = output.precision
+                network_data['example_size'] = examples
```

Comment:
Could it be useful to attach the batch size in here?

Reply:
Definitely need batch size in the report